### ISSUE TITLE
estimated_end_time not showing 10's of hours

### ISSUE THREAD

**Issue by jjaakk** [OllisGit/OctoPrint-DisplayLayerProgress#242]
I have DisplayLayerProgress 1.27.1 installed on OctoPrint 1.7.0, OctoPi 0.18.o - just recently updated to use Python 3.

The estimated_end_time shows finish time as 06:49, when it should be showing 16:49. Why is it not showing the 10's of hours ?
The Dashboard Plugin shows ETA with the correct hour and mins, but the DisplayLayerProgress ETA is missing the 10's value.

Hopefully this is just a time setting in DLP plugin.
Jak.
![Screen Shot 2021-10-18 at 10 26 14 am](https://user-images.githubusercontent.com/12789822/137648896-6c32f753-122e-4130-bd4a-4eb6aebcb460.png)

**Comment by OllisGit** [comment]
Hi @jjaakk,
hmmm..did you changed the Timeformat in the Settings?
![image](https://user-images.githubusercontent.com/24205767/137776595-79876d40-73ad-4618-b74c-2f6f5ef1314e.png)
Please also activate the normal clock in the NavBar as a reference
![image](https://user-images.githubusercontent.com/24205767/137776845-312b880e-84f9-4096-9440-7b30ba4229f5.png)

Btw. the Dashboard-Plugin use the javascript browser date function and DLP use the python server time
DashBoard
![image](https://user-images.githubusercontent.com/24205767/137777401-7f20de46-a589-4691-b13a-9e610ea20883.png)
DLP-Serverside
![image](https://user-images.githubusercontent.com/24205767/137777539-3f12a943-5aec-4381-a85c-b69722bfb762.png)

**Comment by jjaakk** [comment]
I did change the time format to %I:%M. No difference.
I have activated the normal clock as a reference. No difference.
Thanks for the explanation of the two different time mechanisms.

I think I will just keep an eye on the [printtime_left] that I have added into the title bar display, for the moment.
![Screen Shot 2021-10-19 at 8 33 06 am](https://user-images.githubusercontent.com/12789822/137810401-00fcc973-5f98-4b55-a800-3d7a8076e873.png)
